app/models/developer_key.rb

```diff
@@ -101,6 +101,14 @@ def account_name
     account.try(:name)
   end
 
+  def access_token_count
+    self.access_tokens.count
```

Comment:
Count can be quite sad when using postgres, this is a perfect case for using a [counter_cache](http://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache)